Workshop on EDA.ipynb

```diff
@@ -4,26 +4,55 @@
    "cell_type": "markdown",
```

Comment:
<p>Thanks for adding the pull from the URL.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/niwako/garbanzo/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>